app/forms/incomeSources/cease/IncomeSourceEndDateForm.scala

```diff
@@ -30,9 +30,11 @@ import services.DateService
 import java.time.LocalDate
 import javax.inject.Inject
 
-class IncomeSourceEndDateForm @Inject()(val dateService: DateService)(implicit val appConfig: FrontendAppConfig) extends CustomConstraints with FeatureSwitching{
+class IncomeSourceEndDateForm @Inject()(val dateService: DateService)(implicit val appConfig: FrontendAppConfig) extends CustomConstraints with FeatureSwitching {
```

Comment:
I can see a lot of file format validation highlights, can these be addressed?

![Screenshot 2024-07-11 at 11 15 38](https://github.com/hmrc/income-tax-view-change-frontend/assets/165788042/26af200d-793a-477e-9644-5169d9bd4395)

Reply:
The code formatting aspects of this have been addressed, but there are still a couple more warnings - should these be addressed too?